opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/debian/step-2-starting-minion.adoc

```diff
@@ -30,9 +31,10 @@ TIP: Include the `failover:` portion of the broker URL to allow the Minion to re
 
 .Configure the credentials to use when communicating with {opennms-product-name} and exit Karaf shell
 [source]
+[subs=+quotes]
 ----
-opennms:scv-set opennms.http #minion user name minion password
-opennms:scv-set opennms.broker #minion username minion password
+opennms:scv-set opennms.http [red]#minion user name minion password#
```

Comment:
I think "username" should be one word everywhere. It is one argument of this karaf command, not two, so I think having it as "user name" is incorrect.